helm/alfresco-content-services/charts/alfresco-search/requirements.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
```

Comment:
now the requirements are embedded this file can be deleted

Reply:
deleted requirements file